attestation-container/attestation-container.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+
+	"microsoft/attestation-container/attest"
+	pb "microsoft/attestation-container/protobuf"
+
+	"google.golang.org/grpc"
+)
+
+var (
+	port = flag.Int("port", 50051, "The server port")
+)
+
+type server struct {
+	pb.UnimplementedAttestationContainerServer
+}
+
+func (s *server) FetchAttestation(ctx context.Context, in *pb.FetchAttestationRequest) (*pb.FetchAttestationReply, error) {
+	reportData := [attest.REPORT_DATA_SIZE]byte{}
+	if len(in.GetReportData()) > attest.REPORT_DATA_SIZE {
+		return nil, fmt.Errorf("`report_data` needs to be smaller 64 bytes. size: %d bytes", len(in.GetReportData()))
```

Comment:
Can we use `REPORT_DATA_SIZE` in this string, rather than a hard-coded 64?

```suggestion
		return nil, fmt.Errorf("`report_data` needs to be smaller than %d bytes. size: %d bytes", attest.REPORT_DATA_SIZE, len(in.GetReportData()))
```